src/pkg/build/builder_remote.go

```diff
@@ -99,6 +99,14 @@ func (rb *RemoteBuilder) Build(ctx context.Context) (err error) {
 		return err
 	}
 
+	// If we're doing an detached build, print help on how to download the image
+	libraryRefRaw := strings.TrimPrefix(rd.LibraryRef, "library://")
+	if rb.IsDetached {
+		fmt.Printf("Build submitted! Once it is complete, the image can be retrieved by running:\n\n")
+		fmt.Printf("\tsingularity pull --library-uri %v library://%v\n\n", rd.LibraryURL, libraryRefRaw)
```

Comment:
Actually - I missed something. The current flag is `--libraryuri` no hyphen... so the pull command doesn't work. That needs to be fixed... what is the consistent style for flags in Singularity? Hyphen / underscore / none?